packages/config/src/android/Facebook.ts

```diff
@@ -0,0 +1,167 @@
+import { Parser } from 'xml2js';
+import { ExpoConfig } from '../Config.types';
+import { Document } from './Manifest';
+import {
+  getProjectStringsXMLPathAsync,
+  readStringsXMLAsync,
+  setStringItem,
+  writeStringsXMLAsync,
+} from './Strings';
+import { XMLItem } from './Styles';
+
+const facebookSchemeActivity = (scheme: string) => `
+<activity
+    android:name="com.facebook.CustomTabActivity"
+    android:exported="true">
+    <intent-filter>
+        <action android:name="android.intent.action.VIEW" />
+        <category android:name="android.intent.category.DEFAULT" />
+        <category android:name="android.intent.category.BROWSABLE" />
+        <data android:scheme="${scheme}" />
+    </intent-filter>
+</activity>
+`;
+
+export function getFacebookScheme(config: ExpoConfig) {
+  return config.facebookScheme ?? null;
+}
+
+export function getFacebookAppId(config: ExpoConfig) {
+  return config.facebookAppId ?? null;
+}
+
+export function getFacebookDisplayName(config: ExpoConfig) {
+  return config.facebookDisplayName ?? null;
+}
+export function getFacebookAutoInitEnabled(config: ExpoConfig) {
+  return config.hasOwnProperty('facebookAutoInitEnabled') ? config.facebookAutoInitEnabled : null;
```

Comment:
```suggestion
  return config.facebookAutoInitEnabled ?? null;
```

If I'm not mistaken, ?? will only convert undefined or null to null.